### ISSUE TITLE
Erb delimiters not working correctly

### ISSUE THREAD

**Issue by TeriyakiBomb** [ForAzens/prettier-plugin-html-erb#1]
First off, thank you very much for this!

I was testing it out with a scaffold file, after running prettier, it does this:

```
 <% if notice.present? %>
  <p
    class="mb-5 flex rounded-lg bg-green-50 px-3 py-2 font-medium text-green-500"
    id="notice"
  >
    <%= notice %>
  </p>
  <% end %> <% content_for :title, "Tests" %>
```

  
  It puts the end on the same line as the next erb tag

**Comment by TeriyakiBomb** [comment]
I checked in again and I cannot reproduce the issue, sorry. Perhaps it was the older version and I've subsequently updated.

**Comment by TeriyakiBomb** [comment]
Ok, I can now replicate this issue, see attached video

And from package.json:

"devDependencies": {
    "esbuild": "^0.24.0",
    "@4az/prettier-plugin-html-erb": "^0.0.6",
    "@prettier/plugin-ruby": "^4.0.4",
    "prettier": "^3.4.1",
    "prettier-plugin-tailwindcss": "^0.6.9"
  },

https://github.com/user-attachments/assets/111af0e1-1ec4-4e0f-869f-a26d2a843c02

**Comment by doits** [comment]
@TeriyakiBomb This formats correctly for me:

```erb
<div id="tasks" class="flex min-w-full flex-col space-y-2 pt-2">
  <% @tasks.each do |task| %>
    <%= turbo_frame_tag task do %>
      <%= render task %>
    <% end %>
  <% end %>
</div>
```

Can you still reproduce it and provide an example?

**Comment by TeriyakiBomb** [comment]
I'm afraid so, just for my sanity I also tried in a different editor
https://github.com/user-attachments/assets/980c0615-0a08-4c8a-88c6-7d3080f52790

**Comment by doits** [comment]
Maybe you don't have a `.html.erb` filename, but a `.html` filename so the plugin isn't running at all? When I remove `.erb`:

```zsh
echo '
<div id="tasks" class="flex min-w-full flex-col space-y-2 pt-2">
<% @tasks.each do |task| %>
<%= turbo_frame_tag task do %>
<%= render task %>
<% end %>
<% end %>
</div>
' | prettier --stdin-filepath=test.html
<div id="tasks" class="flex min-w-full flex-col space-y-2 pt-2">
  <% @tasks.each do |task| %> <%= turbo_frame_tag task do %> <%= render task %> <% end %> <% end %>
</div>
```

**Comment by TeriyakiBomb** [comment]
No, it is running, I've tried in two editors and my default editor does not have anything configured to format erb, ruby or even HTML without prettier, it's my default formatter. The first video is Panic Nova and the second is ruby mine. These are the correct extension btw.

Might try on a different machine but not sure what difference it'd make.

**Comment by doits** [comment]
Does it work when running prettier in the terminal like in my example? Does it work there (without your editor)?

**Comment by TeriyakiBomb** [comment]
![CleanShot 2025-01-09 at 17 48 37@2x](https://github.com/user-attachments/assets/6efeed59-c936-4a56-a64c-581fa640e647)

Same again!

**Comment by aspirisen** [comment]
I have such error ` Error: No opening statement for closing statement end`
For this code:
```erb
<% product_tiles.sections.each.with_index do |(tiles, header, scrollable, see_all_link), index| %>
  <div>asdasdasdasdasd</div>
<% end %>
```

**Comment by doits** [comment]
@aspirisen I think this is a different problem, you should open a new issue. Your example only works without the braces, so it looks like the parser has problems with them.
